examples/01c-Getting-started-Pytorch.ipynb

```diff
@@ -0,0 +1,553 @@
+{
```

Comment:
<p>We should use a minimal example without NVTabular. We can build one utils function to download and process the dataset?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>removed the dependency on nvtabular completely, having this mirror the previous tensorflow notebook</p>